components/dashboard/src/components/typography/MiddleDot.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC } from "react";
+
+type Props = {
+    className?: string;
+};
+export const MiddleDot: FC<Props> = ({ className }) => {
```

Comment:
**praise**: I just saw this, not sure if it helps but found this component funny.